xsd/NeTEx_publication.xsd

```diff
@@ -285,12 +285,15 @@ Correct COnstraints for PointOnRoute
 		<xsd:keyref name="DataSource_AnyKeyRef" refer="netex:DataSource_AnyVersionedKey">
 			<xsd:selector xpath=".//netex:DataSourceRef | .//netex:DefaultDataSourceRef"/>
 			<xsd:field xpath="@ref"/>
-			<xsd:field xpath="@version"/>
+			<!-- <xsd:field xpath="@version"/> +v2.0-->
```

Comment:
I am sorry I was stupid. But I think we are not there yet. I think for DataSourceRef we must have a version, but I should point to a DataSource key with id + version. Difficult.

Reply:
changed. better?